pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -502,3 +504,45 @@ func Convert_networking_NetworkPolicyList_To_v1beta1_NetworkPolicyList(in *netwo
 	}
 	return nil
 }
+
+func Convert_v1beta1_IngressSpec_To_networking_IngressSpec(in *extensionsv1beta1.IngressSpec, out *networking.IngressSpec, s conversion.Scope) error {
```

Comment:
prefer calling autoConvert_v1beta1_IngressSpec_To_networking_IngressSpec for all the fields that could be converted automatically, then just convert the backend/defaultBackend field manually